Src/BootCamp.Chapter/TextTable.cs

```diff
@@ -32,9 +35,122 @@ public static class TextTable
         /// Padding means how many spaces will a message be wrapped with.
         /// Table itself is made of: "+-" symbols. 
         /// </summary>
+        public const string edgeForTopAndBottom = "-";
+        public const string corner = "+";
+        public const string space = " ";
+        public const string sideBorder = "|";
+
         public static string Build(string message, int padding)
         {
-            return "";
+
+            if (message == "")
+            {
+                return "";
+            }
+            int stringLength;
+            var messageArray = (message.Contains(Environment.NewLine)) ? message.Split(Environment.NewLine) : null;
+            bool isMultiline = messageArray == null ? false : true;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unnecessary Boolean literal(s).](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4953253)